src/org/usfirst/frc4904/robot/commands/BallioIntake.java

```diff
@@ -10,8 +11,9 @@ public BallioIntake() {
 		super("BallioIntake");
```

Comment:
These could probably be combined since the only thing that changes is the values.